src/main/java/net/schmizz/sshj/SSHClient.java

```diff
@@ -440,6 +444,16 @@ public Connection getConnection() {
         return conn;
     }
 
+    /**
+     * Returns the character set used to communicate with the remote machine for certain strings (like paths).
+     *
+     * @return remote character set or {@code null} for default
+     */
+    public Charset getRemoteCharset()
+    {
```

Comment:
Please use the same formatting as the rest of the code, ie. open-curly on end-of-line